default/stringtables/en.txt

```diff
@@ -4628,6 +4628,16 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
+SYSTEM_BLOCKADE_TITLE
+System Blockade
+
+SYSTEM_BLOCKADE_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system, it will establish a system blockade against all empires it is at war with and that currently do not blockade the system. When a blockade is established, [[metertype METER_SUPPLY]] propagation is interrupted for the blockaded empire. Also, if the blockaded empire posesses colonies within the system, the production generated by these colonies can only be used on the planet it is generated on. Additionally, if the blockaded planets are attacked by the incoming fleet, their meter growth (e. g. ground troop regeneration) will be suspended for one turn.
+After a system blockade has been set up, all starlane entries in the blockaded system will be guarded by the blockading force, so if hostile forces enter the system, they can exit only through the starlane entry from which they arrived. However, if more fleets from the blockaded empire arrive by different starlane entries than the first one, these entries will become available as an exit as well. If all armed fleets owned by the blockaded empire are destroyed or leave the system before new fleets from this empire arrive, the previously opened exits will become closed again.
+Allied fleets do not have a direct impact on supply blocks and starlane availability apart from helping with eliminating the blockading empire's forces.
+If fleets from two or more hostile empires arrive at an unblockaded system at the same time, neither such empire will establish a system blockade over each other, although they will establish a blockade against all other hostile empires that arrive at a later time.
```

Comment:
If the fleets arrive at the same time, there will be no system blockade, so other factors will decide who can propagate supply into the system (supply strength, colonies or outpost present, sum of supply sources, distance to nearest supply source). So that means who had supply before will keep it. So I think the description should be correct?